tests/api_test/api_test.cpp

```diff
@@ -447,3 +450,94 @@ TEST_CASE("tailcall_load_test", "[tailcall_load_test]")
 
     bpf_object__close(object);
 }
+
+int
+perform_bind(SOCKET* socket, uint16_t port_number)
+{
+    *socket = INVALID_SOCKET;
```

Comment:
```suggestion
```
line 458 always overwrites it, so no need to initialize it to something else first